### DISCUSSION TITLE
Cannot run python-pytorch-rocm

### DISCUSSION THREAD

**Discussion by fzadikian** [rocm-arch/rocm-arch#967] [Support]
Hello, after installing python-pytorch-rocm in a new Arch Linux installation, I noticed that I cannot run my pytorch-powered scripts as usual. For example, the following [python script](https://paste.debian.net/hidden/017c80c9), outputs [this](https://paste.debian.net/hidden/3610305f), followed by a video card glitch. This is my [rocminfo output](https://paste.debian.net/hidden/abe17a92). I find this weird since on my old Arch, everything (including this script) would work with no issues. Any help would be greatly appreciated. Thanks!

**Comment by jannesklee** [comment]
Hi, I also cannot run pytorch with a [similar error](https://paste.debian.net/hidden/e1a2acf8/) but with even less information. My [rocminfo](https://paste.debian.net/hidden/b4ab95f1/).

**Reply by fzadikian** [reply]
Yeah, I've also got that error message before. I am not sure why ROCm isn't working for me anymore ;(

**Comment by fzadikian** [comment]
Sorry to be necrobumping this but I can't continue working on the python-torchaudio-rocm package otherwise :smiling_face_with_tear:

**Reply by tpkessler** [reply]
We're building the ROCm packages with the default GPU targets set by AMD. Of course, we could add gfx803 to the lists and see if it works but there's no guarantee. gfx803 hasn't been supported a while ago and I have no idea if a recent ROCm release would even build for this target. It would be great if you could locally build one package, say rocsparse or rocblas with gfx803 enabled and see if the test in test.sh passes. Clone this https://gitlab.archlinux.org/archlinux/packaging/packages/rocblas and append gfx803 to AMDGPU_TARGETS. If test.sh passes, we can try to get in all packages. The best way to coordinate this is to register as an Arch Linux Tester, https://wiki.archlinux.org/title/Arch_Testing_Team, and give signoffs to the relevant ROCm packages when they enter [extra-testing].

**Reply by fzadikian** [reply]
Alright, I will do this eventually. However, I do not get why gfx803 stopped working even though it had been tested and put in before (this was way after AMD dropped support for that target). Please see [here](https://github.com/orgs/rocm-arch/discussions/937#discussioncomment-5051036). Could this be because the ROCm stack is not working without modifications to GPU targets anymore (that is, only the pytorch package needed that target added before)?

**Reply by fzadikian** [reply]
Greetings, I have successfully built rocblas with gfx803 appended as a target. Here is the result of my testing:
![](https://github.com/rocm-arch/rocm-arch/assets/13150712/71ee23a9-1217-4c9d-84bd-cc92c856dc21)
What's next @tpkessler? Thanks! :grin:
